1-js/11-async/02-promise-basics/article.md

```diff
@@ -291,7 +291,7 @@ Sometimes, it might be that a promise is already settled when we add a handler t
 In such case, these handlers just run immediately:
 
 ```js run
-// the promise becomes resolved immediately upon creation
+// the promise becomes settled immediately upon creation
```

Comment:
disagree, 'resolve' goes well with use of `resolve` in the article's examples. Maybe 'fulfilled' if trying to use ECMA term.